src/test/groovy/BuildPluginStepTests.groovy

```diff
@@ -180,7 +180,8 @@ class BuildPluginStepTests extends BasePipelineTest {
     assertTrue(helper.callStack.findAll { call ->
       call.methodName == 'node'
     }.any { call ->
-      callArgsToString(call).contains('linux')
+      def args = callArgsToString(call)
+      args.contains('linux') || args.contains('maven')
```

Comment:
I'd say the label should be `maven` only and a new test case might be required when `useAci: true`.